app/v2/controllers/V2MovementsController.scala

```diff
@@ -783,7 +992,19 @@ class V2MovementsControllerImpl @Inject() (
         Some(movementType),
         Some(messageType)
       )
-      _ = pushNotificationsService.update(movementId)
+      _ = pushNotificationsService.update(movementId).leftMap {
+        err =>
+          auditService.auditStatusEvent(
+            PushNotificationUpdateFailed,
+            None,
+            Some(movementId),
+            Some(updateMovementResponse.messageId),
+            Some(request.eoriNumber),
+            Some(movementType),
+            None
```

Comment:
At this point of time we will be having messageType available so please pass that.